grapher/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -232,18 +249,45 @@ export class StackedDiscreteBarChart
             }))
             .filter((item) => item.bars.length)
 
-        if (this.manager.isRelativeMode) {
-            // TODO: This is more of a stopgap to prevent the chart from being super jumpy in
-            // relative mode. Once we have an option to sort by a specific metric, that'll help.
-            // Until then, we're sorting by label to prevent any jumping.
-            return sortBy(items, (item) => item.label)
-        } else {
-            return sortBy(items, (item) => {
-                const lastPoint = last(item.bars)?.point
-                if (!lastPoint) return 0
-                return lastPoint.valueOffset + lastPoint.value
-            }).reverse()
+        return items
+    }
+
+    @computed get sortedItems(): readonly Item[] {
+        let sortState = { ...this.sortConfig }
+
+        if (this.manager.isRelativeMode && sortState.sortBy === SortBy.total) {
+            // In relative mode, where the values for every entity sum up to 100%, sorting by total
+            // doesn't make sense. It's also jumpy because of some rounding errors. For this reason,
+            // we sort by entity name instead.
+            sortState = {
+                ...sortState,
+                sortBy: SortBy.entityName,
+                sortOrder: SortOrder.asc,
+            }
```

Comment:
Does this logic maybe belong to `@computed get sortConfig()`? In case anything else accesses sort config.

Reply:
Good point! I now moved it into `Grapher` entirely.
At first I thought it might make sense to have different default sort configs per chart, but that's difficult to do correctly with the Admin, and sorting by total value seems like it's the most sensible default for now anyway.